packages/venia-concept/src/reducers/checkout.js

```diff
@@ -1,70 +1,85 @@
+import { handleActions } from 'redux-actions';
+
+import actions from 'src/actions/checkout';
+
+export const name = 'checkout';
+
 const initialState = {
-    shippingInformation: false,
-    status: 'READY',
-    subflow: null
+    editing: null,
+    step: 'cart',
+    submitting: false,
+    valid: false
 };
```

Comment:
Storing validity here as a simple boolean, `valid`, is definitely an oversimplification. This will get much more complex as we start to track client- and server-side validation of various parts of the form and the workflow.

Reply:
Yeah, I think that reducers should be autogenerated at the same time as actions, see comment above.